pom.xml

```diff
@@ -390,19 +390,26 @@ under the License.
 			<dependency>
 				<groupId>com.fasterxml.jackson.core</groupId>
 				<artifactId>jackson-annotations</artifactId>
-				<version>2.10.1</version>
+				<version>${jackson.version}</version>
 			</dependency>
 
 			<dependency>
 				<groupId>com.fasterxml.jackson.core</groupId>
 				<artifactId>jackson-databind</artifactId>
-				<version>2.10.1</version>
+				<version>${jackson.version}</version>
 			</dependency>
 
 			<dependency>
 				<groupId>com.fasterxml.jackson.core</groupId>
 				<artifactId>jackson-core</artifactId>
-				<version>2.10.1</version>
+				<version>${jackson.version}</version>
+			</dependency>
+
+			<!-- https://issues.apache.org/jira/browse/FLINK-15868 -->
+			<dependency>
+				<groupId>com.fasterxml.jackson.dataformat</groupId>
+				<artifactId>jackson-dataformat-cbor</artifactId>
+				<version>${jackson.version}</version>
 			</dependency>
```

Comment:
Do we really want to change the `jackson-dataformat-cbor` for all dependencies which rely on it? It would invalidate the testing for all these components.

On the other hand I guess that the functionality used by Kinesis should then also be broken for all the other modules.

Reply:
What about bumping other jackson dependencies such as `jackson-dataformat-smile`, `jackson-dataformat-yaml`, `jackson-module-jaxb-annotations`? Could they also be affected by the bumping the other dependencies to `2.10.1`?